R/get_population.R

```diff
@@ -22,7 +22,7 @@
 #'
 #' @examples
 #' \dontrun{
-#'  get_population(geo_type = "region")[]
+#'  get_population(geo_type = "region")
```

Comment:
I think this should have the [] at the end so that it will print on the screen for the user who wants to try it. I had an APDE staff person ask me why it didn't produce output previously.